platform/templates/secrets/ldap.yaml

```diff
@@ -1,13 +1,13 @@
-{{- if .Values.ldap.enabled }}
+{{- if .Values.openldap.enabled }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: ldap
 type: Opaque
 data:
-  host: bGRhcA== #ldap
-  base_dn: ZGM9ZXhhbXBsZSxkYz1vcmc= #dc=example,dc=org
-  user: YWRtaW51c2VybmFtZT1hZG1pbg== #adminusername=admin
-  password: cGFzc3dvcmQ9YWRtaW4= #password=admin
-  environment_ou: dGVzdA== #test
-{{- end }}
+  host: bGRhcC1vcGVubGRhcA== # ldap-openldap
```

Comment:
We should consider making this section more configurable and secure. Probably out of scope here.